### ISSUE TITLE
Compilation errors: libmbed.a libcc_310_core.a libcc_310_ext.a libcc_310_trng.a not found

### ISSUE THREAD

**Issue by carlosperate** [googlecreativelab/tf4micro-motion-kit#3]
Arduino IDE version: 1.8.12 (v2-beta also has the same issue).
Board: Arduino Nano 33 BLE Sense
Arduino Core: Arduino Mbed OS Nano Boards 2.1.0

Tried the Arduino_TensorFlowLite library version `2.4.0-ALPHA` and `2.1.1-ALPHA-precompiled` and getting the same compilation errors:

```
WARNING: library ArduinoBLE claims to run on samd, megaavr, mbed, apollo3 architecture(s) and may be incompatible with your current board which runs on mbed_nano architecture(s).
arm-none-eabi-g++: error: ../../../../../../Users/carlos/Library/Arduino15/packages/arduino/hardware/mbed_nano/2.1.0/variants/ARDUINO_NANO33BLE/libs/libmbed.a: No such file or directory
arm-none-eabi-g++: error: ../../../../../../Users/carlos/Library/Arduino15/packages/arduino/hardware/mbed_nano/2.1.0/variants/ARDUINO_NANO33BLE/libs/libcc_310_core.a: No such file or directory
arm-none-eabi-g++: error: ../../../../../../Users/carlos/Library/Arduino15/packages/arduino/hardware/mbed_nano/2.1.0/variants/ARDUINO_NANO33BLE/libs/libcc_310_ext.a: No such file or directory
arm-none-eabi-g++: error: ../../../../../../Users/carlos/Library/Arduino15/packages/arduino/hardware/mbed_nano/2.1.0/variants/ARDUINO_NANO33BLE/libs/libcc_310_trng.a: No such file or directory
exit status 1
Error compiling for board Arduino Nano 33 BLE.
```

**Comment by carlosperate** [comment]
Also tried the deprecated "Arduino Mbed OS boards" core and getting the same result.

With both cores the simple blinky example compiles and flashes correctly.

This looks like an issue in https://github.com/arduino/ArduinoCore-mbed/ I'll search the issue tracker and open an issue if not reported already.

**Comment by carlosperate** [comment]
Turned out to be an issue with the Arduino IDE I was using (1.8.12). This was fixed updating to the latest version 1.8.15.

Then I had a small issue not finding some debug symbols, but that was easily resolved by updating the Arduino_TensorFlowLite from 2.1 to 2.4-alpha.
